include/motorError.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _MOTORERROR_H_
+#define _MOTORERROR_H_
+
+#include "telemetry.h"
+#include <string>
+
+enum ErrorType {None, Error1, Error3, Error4, Error5, Error6, Error8, Error9, Unknown};
+
+extern ErrorType errorType;
+
+void startErrorInterpretation();
+const char * errorString(ErrorType errorType);
```

Comment:
Short comment for both would be nice. You do a great job describing what each function does in the source file, but this would let someone use the functions without doing any digging.